### ISSUE TITLE
New docker installation fails to start

### ISSUE THREAD

**Issue by brandon-dacrib** [rogerfar/rdt-client#200]
using current latest under via docker-compose within ubuntu 20 the application fails to start with the following repeating in the log file 
```Starting rtdclient
[21:36:05 WRN] Using an in-memory repository. Keys will not be persisted to storage.
[21:36:05 WRN] Neither user profile nor HKLM registry available. Using an ephemeral key repository. Protected data will be unavailable when application exits.
[21:36:05 WRN] No XML encryptor configured. Key {eecab321-7f4c-4d1d-87aa-ff91b289ad4f} may be persisted to storage in unencrypted form.
[21:36:05 WRN] Starting host on version 2.0.19.0
[21:36:06 ERR] Failed executing DbCommand (25ms) [Parameters=[], CommandType='Text', CommandTimeout='30']
SELECT COUNT(*) FROM "sqlite_master" WHERE "name" = '__EFMigrationsHistory' AND "type" = 'table';
[21:36:06 ERR] BackgroundService failed
System.Threading.Tasks.TaskCanceledException: A task was canceled.
   at RdtClient.Service.BackgroundServices.ProviderUpdater.ExecuteAsync(CancellationToken stoppingToken) in /appserver/server/RdtClient.Service/BackgroundServices/ProviderUpdater.cs:line 25
   at Microsoft.Extensions.Hosting.Internal.Host.TryExecuteBackgroundServiceAsync(BackgroundService backgroundService)```


my relevant docker compose   rdtclient:
    image: rogerfar/rdtclient
    container_name: rdtclient
    environment:
      - PUID=1000
      - PGID=100
      - TZ=America/New_York
    volumes:
      - /config/rdtclient/data/db:/data/db
      - /media/rdtclient/downloads:/data/downloads
    networks:
      - doris
    logging:
       driver: json-file
       options:
          max-size: 10m
    ports:
      - 6500:6500
    labels:
      - traefik.frontend.rule=Host:rdtclient.dacrib.net
      - traefik.frontend.entryPoints=http,https
    restart: unless-stopped

**Comment by brandon-dacrib** [comment]
I figured maybe it was permissions problem so I tried just chmod -R ugo+rwx the relevant directories to no avail.

**Comment by brandon-dacrib** [comment]
Turns out this was some strange disk i/o error with unraid mounted volumes and sqlite3. When I installed sqlite3 and tried to run `SELECT COUNT(*) FROM "sqlite_master" WHERE "name" = '__EFMigrationsHistory' AND "type" = 'table';` against the sqlite3 db it failed with a disk i/o error. Then I copied `rdtclient.db` to /tmp and attempted the same and it was fine. Maybe some posix mis-compliance that I don't understand with unraid. In either case moving the file off of the unraid mounted volume worked. Leaving this here in case anyone ever runs into it.
